common/app/model/dotcomrendering/pageElements/CalloutExtraction.scala

```diff
@@ -188,6 +193,8 @@ object CalloutExtraction {
       tagName <- (campaign \ "fields" \ "tagName").asOpt[String]
       formFields1 <- (campaign \ "fields" \ "formFields").asOpt[JsArray]
     } yield {
+      val contacts = (campaign \ "fields" \ "contacts").asOpt[Seq[Contact]]
```

Comment:
Very interesting that we don't need to do the parsing with the primitive values and `Contact` can be used directly. Whatever example I find online with `asOpt` and also within frontend, all are using it with the innermost primitive fields such as String, Long, etc. But what you did seems to work perfectly 👍